src/layout/tooltip/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function Tooltip() {
```

Comment:
```suggestion
export default function Tooltip(props: {
    children: JSX.Element,
    padding?: number,
}) {
```

Basic props for the tooltip so it can be customizable for different use cases